lms/djangoapps/branding/api.py

```diff
@@ -171,6 +172,13 @@ def _footer_social_links():
 
 def _footer_connect_links():
     """Return the connect links to display in the footer. """
+    domain_name = Site.objects.get_current().domain
+
+    if domain_name == "edx.org":
+        contact_us_link = "https://courses.edx.org/support/contact_us"
```

Comment:
Not sure, if we can hard code these links. @staubina Could you please review this.

Reply:
We should get the root from the settings file.  Let me know if you are unable to find an example of this in the code.  In general we should avoid hard coding any URL as the community will use a different root URL.